test/tests/functional/pbs_qstat.py

```diff
@@ -68,3 +68,8 @@ def test_qstat_req_server(self):
                  " from %s@%s" % (str(TEST_USER), self.server.client)
         self.server1.log_match(msg=expmsg, existence=False, max_attempts=5)
         self.server2.log_match(msg=expmsg)
+
+    def tearDown(self):
+        a = {'PBS_SERVER': '@%s' % self.server1.hostname}
```

Comment:
this will set PBS_SERVER to server1.hostname regardless of the previous value.
You need to save the previous value and use it here. Also, attach the test logs.